src/middleware/uploadFile.js

```diff
@@ -0,0 +1,22 @@
+const multer = require('multer');
+const { BadRequestError } = require('../errors');
+
+const MAX_FILE_SIZE = 1 * 1024 * 1024;
+const ALLOWED_FILE_TYPES = ['image/jpeg', 'image/png', 'image/webp', 'image/bmp'];
+
+const storage = multer.memoryStorage();
+
```

Comment:
Food for thought: we don't reject corrupted files or any specific encodings, which is totally fine for this application, but I want you know have that in your brains.